Purchases/Public/RCPurchases.h

```diff
@@ -101,7 +101,7 @@ NS_SWIFT_NAME(Purchases)
 /**
  Configures an instance of the Purchases SDK with a specified API key. The instance will be set as a singleton. You should access the singleton instance using [RCPurchases sharedPurchases]
 
- @note Use this initializer if your app does not have an account system. `RCPurchases` will generate a unique identifier for the current device and persist it to `NSUserDefaults`. This also affects the behavior of `restoreTransactionsForAppStoreAccount`.
+ @note Use this initializer if your app does not have an account system. `RCPurchases` will generate a unique identifier for the current device and persist it to `NSUserDefaults`. This also affects the behavior of `restoreTransactionsWithCompletionBlock`.
```

Comment:
while we're at it, could you add line breaks so that this doesn't scroll horizontally? I've been using ~120 chars as the limit